devel/rtags/Portfile

```diff
@@ -4,30 +4,26 @@ PortSystem          1.0
 PortGroup           cmake 1.0
 PortGroup           github 1.0
 
-github.setup        Andersbakken rtags 2.21 v
-revision            1
+github.setup        Andersbakken rtags 2.33 v
+revision            0
+github.tarball_from releases
 categories          devel
 platforms           darwin
 license             GPL-3+
 
+checksums           rmd160  156ccfa3adbb7e94368a1b52b9058a7aa7082db3 \
+                    sha256  8cc4e1a713275510cbe1f5f413d3e6a3d943e12b01ede573d1dba9b3be649a75 \
+                    size    1606535
+
 maintainers         nomaintainer
 
 description         A c/c++ client/server indexer for c/c++/objc\[++\] with \
                     integration for Emacs based on clang
 long_description    ${description}
 
 # just pick the newest version of clang (make sure to update below)
-depends_lib-append  port:clang-6.0
-
-# Fetch from git instead of distfile because it needs submodules
-fetch.type          git
-
-post-fetch {
-    system -W "${worksrcpath}" "git submodule update --init"
-}
-
-
-depends_lib-append  port:zlib
+depends_lib         port:clang-8.0 \
```

Comment:
Is this really the correct dependency ? The command llvm-config used below is provided by the llvm-8.0 port, not clang.

Reply:
Yes, it is. rtags links to libclang.dylib, which is part of the clang-8.0 port. llvm-config is used to find it:

https://github.com/Andersbakken/rtags/blob/master/cmake/FindLibClang.cmake